<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodeBrowserDockWidget</class>
 <widget class="QDockWidget" name="CodeBrowserDockWidget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="features">
   <set>QDockWidget::AllDockWidgetFeatures</set>
  </property>
  <property name="windowTitle">
   <string>Assembly Browser</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTableView" name="tableView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ContiguousSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <property name="showGrid">
       <bool>false</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::NoPen</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>26</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>50</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>23</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>23</number>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionBreak_on_CPU_execution_here">
   <property name="text">
    <string>Break on CPU execution here</string>
   </property>
   <property name="toolTip">
    <string>Break on CPU execution here</string>
   </property>
  </action>
  <action name="actionRun_to_here">
   <property name="text">
    <string>Run to here</string>
   </property>
   <property name="toolTip">
    <string>Run to here</string>
   </property>
  </action>
  <action name="actionDisable_breakpoint">
   <property name="text">
    <string>Disable this breakpoint</string>
   </property>
   <property name="toolTip">
    <string>Disable this breakpoint</string>
   </property>
  </action>
  <action name="actionRemove_breakpoint">
   <property name="text">
    <string>Remove this breakpoint</string>
   </property>
   <property name="toolTip">
    <string>Remove this breakpoint</string>
   </property>
  </action>
  <action name="actionEnable_breakpoint">
   <property name="text">
    <string>Enable this breakpoint</string>
   </property>
   <property name="toolTip">
    <string>Enable this breakpoint</string>
   </property>
  </action>
  <action name="actionStart_marker_here">
   <property name="text">
    <string>Set Visualization START Marker here</string>
   </property>
   <property name="toolTip">
    <string>Set Visualization START Marker here</string>
   </property>
  </action>
  <action name="actionEnd_marker_here">
   <property name="text">
    <string>Set Visualization END Marker here</string>
   </property>
   <property name="toolTip">
    <string>Set Visualization END Marker here</string>
   </property>
  </action>
  <action name="actionClear_marker">
   <property name="text">
    <string>Clear Visualization Markers</string>
   </property>
   <property name="toolTip">
    <string>Clear Visualization Markers</string>
   </property>
  </action>
  <action name="actionGo_to_Source">
   <property name="text">
    <string>Go to Source</string>
   </property>
   <property name="toolTip">
    <string>Go to Source</string>
   </property>
  </action>
  <action name="actionStart_End_marker_here">
   <property name="text">
    <string>Set Visualization Marker on selected region</string>
   </property>
   <property name="toolTip">
    <string>Set Visualization Marker on selected region</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
